cpp/src/link_analysis/pagerank.cu

```diff
@@ -276,14 +277,17 @@ gdf_error gdf_pagerank(gdf_graph *graph, gdf_column *pagerank,
   //
   //  If csr doesn't exist, create it.  Then check type to make sure it is 32-bit.
   //
-  GDF_REQUIRE(graph->adjList != nullptr || graph->edgeList != nullptr, GDF_INVALID_API_CALL);
+  /*GDF_REQUIRE(graph->adjList != nullptr || graph->edgeList != nullptr, GDF_INVALID_API_CALL);

```

Comment:
-> Remove adding adjacency list since transposed adjacency list is already added/called in python wrapper and this data is required by and passed to cugraph::HT_matrix_csc_coo and cugraph::pagerank calls.
-> Remove adjacency list checks. The offsets and indices are ensured to be int32 since it is checked and typecasted while adding the graph itself (within add_edge_list and add_adj_list functions in graph_wrapper.pyx)

Reply:
lgtm

Reply:
it seems we should have a GDF_REQUIRE(gdf_G->transposedAdjList, GDF_INVALID_API_CALL);